src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -1593,6 +1693,11 @@ public class GetBigBookOperationCallableLongRunningCallableWap {
  * limitations under the License.
  */
 // DO NOT EDIT! This is a generated sample ("LongRunningCallable",  "wap2")
+// sample-metadata:
+//   title: GetBigBook: 'War and Peace'
+//   description: Testing resource name overlap
+//   usage: gradle run -PmainClass=com.google.example.examples.library.v1.GetBigBookOperationCallableLongRunningCallableWap2 [--args='[--shelf Novel] [--big_book_name "War and Peace"]']
```

Comment:
Do we want the usage to refer to `gradle run` or simply to `java`? It seems like the latter is more general and also how we're setting the invocation for the sample tests.

Reply:
I have this concern too.

The good thing about `gradle run` is that we generate a `build.gradle` for the sample package, so the command would just work on the generated samples.

The main concern about `java` is that depending on how people compiles the java code (`javac` or mvn/gradle`) the compiled jar might be in different directories, so it's hard to provide a command that just works.

Probably this is a question for @beccasaurus ?

Reply:
Hmm, I see. Given that we generate the gradle files, maybe leaving it to refer to gradle makes sense. (I'd still prefer to have it more generic, but it seems that's likely to work less than simply referring to our gradle config)